### DISCUSSION TITLE
Publish project as Azure Container Apps Job

### DISCUSSION THREAD

**Discussion by christiannagel** [microsoft/aspire#3556] [Q&A] [answered]
What's the current best way to publish a project as Azure Container Apps Job?

With a project that's triggered on storage queues or event hub, instead of an Azure Container App, an Azure Container Apps Job should be published. Would this be possible by assigning a Bicep template to the app model?

I didn't find it in the roadmap to directly support Azure Container Apps Jobs.

**Comment by davidfowl** [comment, marked as answer]
> What's the current best way to publish a project as Azure Container Apps Job?

There's no support for this via aspire. You'd need to mutate the generated bicep using azd infra synth.

Customizing the container app publish outside of infra synth will happen post GA.

**Comment by christiannagel** [comment]
This is now available as experimental feature: https://learn.microsoft.com/en-us/dotnet/aspire/azure/container-app-jobs
